apps/cms/pages/templates/a4_candy_cms_pages/blocks/col_cta_block.html

```diff
@@ -10,10 +10,11 @@
             <div class="block-col_cta-content">
                 <a href="{{ column.link }}" class="btn btn--full btn--transparent">{{ column.link_text }}</a>
             </div>
-            {% else %}
+            {% elif not column.body and column.link %}
```

Comment:
`{% elif column.link %}` is enough here, because the other one is taken care of with the ones above. (It will only get here if the conditions above are not True.)